### PR TITLE
feat: [IA-673] Added versions history permission field

### PR DESCRIPTION
**Pull request by dgopsq**
## Short description
This PR is going to add a permission field displayed to the user for the `versionsHistory` field on Zendesk. This permission field will replace the old `Version of the app in use`. The current app version will be sent anyway for backward compatibility.

image

## List of changes proposed in this pull request
- Removed the `Version of the app in use` field
- Added the `versionsHistory` field
- For backward compatibility the current app version of the app is going to be sent anyway

## How to test
1. Open the contextual help and start writing a ticket
2. The `App version history` permission field should be displayed instead of the old `Version of the app in use`
3. Opening the ticket on Zendesk both the app version and the history should be available in the dashboard. This behaviour should be unchanged from the current app version.

### CODE REVIEW

**Review by thisisjp — APPROVED** [review]

File: ts/features/zendesk/screens/ZendeskAskPermissions.tsx [hunk 1]
```diff
@@ -210,7 +209,6 @@ const ZendeskAskPermissions = (props: Props) => {
     deviceDescription: `${getModel()} · ${
       isIos ? "iOS" : "Android"
     } · ${getSystemVersion()}`,
-    currentVersion: getAppVersion(),
```
Comment (emiliopavia) [inline ts/features/zendesk/screens/ZendeskAskPermissions.tsx:213]: Is there a reason why this has been moved out? Just curious.

**Review by emiliopavia — COMMENTED** [review]
I just left an inline question.

File: ts/features/zendesk/screens/ZendeskAskPermissions.tsx (hunk 1 above)
Comment (dgopsq) [reply ts/features/zendesk/screens/ZendeskAskPermissions.tsx:213]: As far as I can see, if I leave that there I have to add an item inside `itemProps` with a `value` and a `zendeskId` to match the field in the Zendesk dashboard, and doing that would basically show the old `Version of the app in use` field to the user. What I did is: replace that old field with a new one related to the history of versions, sending anyway the current version "outside" of the `itemProps` object for backward compatibility!

If I'm missing something or you see a better way to do that (maybe with less changes too) please let me know 🙂

Comment (emiliopavia) [reply ts/features/zendesk/screens/ZendeskAskPermissions.tsx:213]: Nope, I guess it's ok.

**Review by emiliopavia — APPROVED** [review]

**Pull request merged**